codedeploy_role_document.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Sid": "",
+    "Effect": "Allow",
+    "Principal": {
+      "Service": [
+        "codedeploy.us-east-1.amazonaws.com",
+        "codedeploy.eu-west-1.amazonaws.com",
+        "codedeploy.us-west-2.amazonaws.com",
```

Comment:
Not sure - since our instances, s3 buckets, etc are in this region, I assumed that it was required.